mllib-local/src/main/scala/org/apache/spark/ml/linalg/BLAS.scala

```diff
@@ -20,22 +20,42 @@ package org.apache.spark.ml.linalg
 import com.github.fommil.netlib.{BLAS => NetlibBLAS, F2jBLAS}
 import com.github.fommil.netlib.BLAS.{getInstance => NativeBLAS}
 
+import org.apache.spark.SparkConf
+import org.apache.spark.internal.config.NATIVE_L1_THRESHOLD
+
 /**
  * BLAS routines for MLlib's vectors and matrices.
  */
 private[spark] object BLAS extends Serializable {
 
   @transient private var _f2jBLAS: NetlibBLAS = _
   @transient private var _nativeBLAS: NetlibBLAS = _
+  private val nativeL1Threshold =
+    new SparkConf().getInt(NATIVE_L1_THRESHOLD.key, 256)
```

Comment:
You should be able to access the value from the ConfigEntry directly, and not repeat the default